Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:22.04
```

Comment:
Nit: did we try using a python base repo?

Reply:
I did not - Since this is also running the tower-cli in the container (Not just python). I ended up being this base image; Since it worked, I didn't go back to change it